VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -76,6 +81,16 @@ export const HomeScreen: FC<HomeScreenProps> = ({ navigation }) => {
       <Box flex={1} justifyContent="flex-start">
         <EncourageUpdateAlert />
         <Nametag />
+        {upcomingAppointmentsCount && (
+          <Box mx={theme.dimensions.gutter} mb={theme.dimensions.condensedMarginBetween}>
+            <LargeNavButton
+              title={`${t('appointments')}`}
+              subText={`(${upcomingAppointmentsCount} ${t('upcoming')})`}
+              onPress={() => Linking.openURL('vamobile://appointments')}
```

Comment:
Unfortunately, the appointments screen needs to be opened via a deep link to ensure back navigation is handled properly. This will most likely be the case with the other home screen modules.